deployments/helm/nvidia-device-plugin/templates/daemonset-gfd.yml

```diff
@@ -199,6 +201,10 @@ spec:
         - name: host-sys
           hostPath:
             path: "/sys"
+        - name: nvidia-imex-dir
+          type: DirectoryOrCreate
+          hostPath:
+            path: {{ include "nvidia-device-plugin.filepathJoin" (list "/" .Values.nvidiaDriverRoot "/etc/nvidia-imex") }}
```

Comment:
```suggestion
            path: {{ clean ( join "/" ( list .Values.nvidiaDriverRoot "/etc/nvidia-imex" ) ) | quote }}
```

Reply:
you mean, dropping the helper function?

Reply:
Is this change going to do what we want? The normal join is not a `filepath.Join()` so I'm not sure if it will treat redundant / missing slashes correctly.

Reply:
did `{{ clean ( join "/" ( list "/" .Values.nvidiaDriverRoot "/etc/nvidia-imex" ) ) | quote }}`
tested with values:
- null
- /
- /proc/nvidia
- proc/nvidia
- /proc/nvidia/